### DISCUSSION TITLE
Pyspark for ABRiS bridge to confluent avro

### DISCUSSION THREAD

**Discussion by kothapet** [AbsaOSS/ABRiS#303] [Q&A] [answered]
Hello, is there are a pyspark version for ABRiS bridge to confluent avro?

**Comment by cerveada** [comment, marked as answer]
Hi, sorry, I don't really understand to your question.

Here is a link to python documentation:
https://github.com/AbsaOSS/ABRiS/blob/master/documentation/python-documentation.md

maybe it will help.

**Comment by kothapet** [comment]
Thanks @cerveada you answered it. I found it too.. :)
